content/manuals/subscription/core-subscription/details.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: Docker Core subscriptions and features
+linkTitle: Subscriptions and features
 description: Learn about Docker Core subscription tiers and their features
 keywords: subscription, personal, pro, team, business, features, docker core subscription, docker core
 aliases:
 - /subscription/details/
+weight: 310
```

Comment:
Weights are relative to the current section. The weight of a section page determines the placement of that section in the parent section. The weight of a single page determines the placement of that page in its `.CurrentSection`.